procrastinate/worker.py

```diff
@@ -153,21 +157,23 @@ def load_task(self, task_name) -> tasks.Task:
             log_level = logging.INFO
             exc_info = False
         finally:
-            end_time = log_context["end_timestamp"] = time.time()
-            log_context["duration_seconds"] = end_time - start_time
-            extra = {"action": log_action, "job": log_context, "result": task_result}
-            logger.log(log_level, log_title, extra=extra, exc_info=exc_info)
+            end_time = context["end_timestamp"] = time.time()
+            context["duration_seconds"] = end_time - start_time
+            extra = {"action": log_action, "job": context, "result": task_result}
+            seconds = round(context['duration_seconds'], 3)
+            ftext = f"{log_title} on job {context['call_string']} in {seconds}"
```

Comment:
```suggestion
            ftext = f"{log_title} on job {context['call_string']} in {context['duration_seconds']:.3f}"
```
and you won't need the round anymore.

When in doubt, check https://pyformat.info/

Reply:
Yes ! I forgot, I was maybe a bit lazy last week sorry